### DISCUSSION TITLE
uv_prepare_init method

### DISCUSSION THREAD

**Discussion by sinkhaha** [libuv/libuv#3924] [Q&A] [answered]
I'm new to libuv. Where is the source code for the method uv_prepare_init in uv  v1.43.0?

**Comment by santigimeno** [comment, marked as answer]
They're defined [here](https://github.com/libuv/libuv/blob/v1.x/src/unix/loop-watcher.c) and [here](https://github.com/libuv/libuv/blob/v1.x/src/win/loop-watcher.c).

**Reply by sinkhaha** [reply]
thank you!
